generator/src/test/java/com/scottlogic/deg/generator/cucumber/If.feature

```diff
@@ -1583,7 +1583,7 @@ Scenario: Running an if request that contains a contradictory shorterThan constr
        | foo    | bar    |
        | "dddd" | "4444" |
 
-@ignore
+@ignore #issue 246
```

Comment:
I think this has been resolved now? If not can you create a new issue as #246 is closed.

Reply:
I am altering this as part of ticket #501 - is that OK?

Reply:
Yes, fine